files/files.go

```diff
@@ -114,6 +114,36 @@ func (c *Content) Sys() interface{} {
 	return nil
 }
 
+type EmptyFolder struct {
+	Path  string      `yaml:"path,omitempty"`
+	Owner string      `yaml:"owner,omitempty"`
+	Group string      `yaml:"group"`
+	Mode  os.FileMode `yaml:"mode,omitempty"`
+	MTime time.Time   `yaml:"mtime,omitempty"`
+}
+
+type EmptyFolders []*EmptyFolder
+
+func (f *EmptyFolder) WithFolderInfoDefaults() *EmptyFolder {
```

Comment:
this should probably happen on the "global" `WithDefaults` function... iterate the empty folders list and set defaults as needed...

Reply:
Oh,Sorry...I missed the function `WithDefaults `. I will reconsider that and update.